nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -450,7 +450,9 @@ in {
         else { PORT = toString(cfg.streamingPort); }
       );
       serviceConfig = {
-        ExecStart = "${pkgs.nodejs-slim}/bin/node streaming";
+        # With nodejs v14 the streaming endpoint breaks. Need migrate to uWebSockets.js or similar.
+        # https://github.com/tootsuite/mastodon/issues/15184
+        ExecStart = "${pkgs.nodejs-slim-12_x}/bin/node streaming";
```

Comment:
Good catch, but I don't like the duplication of both comments and code here. I see myself updating nodejs in the future and forgetting about one of the occurences. Maybe we could add a script like `run-streaming.sh` in the package, that executes `streaming` with the current nodejs version? This way the wouldn't need to reference nodejs in the module.

Reply:
Remove dublicate.

> Maybe we could add a script like `run-streaming.sh` in the package, that executes `streaming` with the current nodejs version? This way the wouldn't need to reference nodejs in the module.

I don't know how to realise this variant.

Reply:
I pushed a commit to your branch. Feel free to squash it into yours, so this PR contains only one commit. Then I'd merge this.

Reply:
Ok. Rebsed PR.
Thanks!